examples/va/hci/dataplane-post-ceph.md

```diff
@@ -23,22 +23,22 @@ vi service-values.yaml
 The ceph sections of [values.yaml](values.yaml) should have values like this.
 ```yaml
 data:
+  ceph:
+    conf: $CONF
+    keyring: $KEY
+  nova:
     ceph:
-        conf: $CONF
-        keyring: $KEY
-    nova:
-      ceph:
-        conf:
-            conf: |
-              [libvirt]
-              images_type=rbd
-              images_rbd_pool=vms
-              images_rbd_ceph_conf=/etc/ceph/ceph.conf
-              images_rbd_glance_store_name=default_backend
-              images_rbd_glance_copy_poll_interval=15
-              images_rbd_glance_copy_timeout=600
-              rbd_user=openstack
-              rbd_secret_uuid=$FSID
+      conf:
+        conf: |
```

Comment:
```yaml
conf:
  conf: |
```
should be:
```
conf: |
```
The original had this problem too.

https://github.com/openstack-k8s-operators/architecture/blob/main/examples/va/hci/values.yaml#L23

Reply:
Ack, done